### DISCUSSION TITLE
@testable import ClientRuntime - fatal error

### DISCUSSION THREAD

**Discussion by djpara** [awslabs/aws-sdk-swift#1354] [Q&A] [answered]
Hey all,

I'm running server-side swift on a ubuntu EC2 instance and I'm trying to compile and run my app, but I'm coming across this error. It looks like there's an `@testable import` in production code? I've never seen that before -- wondering if this import was intentional? Anyway, it's not letting me build my application. Running version 0.35.0.

<img width="578" alt="image" src="https://github.com/awslabs/aws-sdk-swift/assets/19669841/a5015d5f-ccb8-4875-8921-182f2e15c520">

Thanks in advance!

**Comment by jbelkins** [comment, marked as answer]
Thanks for your report, and sorry for the trouble.

I'm not sure how that `@testable` got there, but it is not needed and will be removed in our next release.

**Reply by djpara** [reply]
Thanks for the super-quick response!

**Reply by jbelkins** [reply]
May I ask, were you building with Release configuration to get that error?

I'd like to determine why we didn't see it in our own pre-release testing.

**Reply by djpara** [reply]
Yes, release

**Reply by jbelkins** [reply]
(By the way, the most convenient way to get around this for now might be to require AWS SDK for Swift 0.34.0.)

**Reply by jbelkins** [reply]
@djpara Please try our latest release (0.36.0) and see if it builds without the error you described above.

**Reply by sichanyoo** [reply]
@djpara Below is link to release notes: 
https://github.com/awslabs/aws-sdk-swift/releases/tag/0.36.0.

**Reply by djpara** [reply]
Yup! Worked great!
